modules/datastore/modules/datastore_mysql_import/src/Service/MysqlImport.php

```diff
@@ -89,15 +89,9 @@ protected function runIt() {
    */
   protected function getColsFromFile(string $file_path, string $delimiter): array {
 
-    // Ensure the "auto_detect_line_endings" ini setting is enabled before
-    // openning the file to ensure Mac style EOL characters are detected.
-    $old_ini = ini_set('auto_detect_line_endings', '1');
```

Comment:
Maybe make a note in the issue somewhere that this changes compatibility with \r line endings. https://php.watch/versions/8.1/auto_detect_line_endings-ini-deprecated

Reply:
Thanks @paul-m ! I don't see a change log file. Where would I update the release notes as part of this PR?

Reply:
Janette added a mention in the draft release notes for the next DKAN release here: https://github.com/GetDKAN/dkan/releases

Reply:
Will document in the release notes.

Reply:
Confirmed @janette drafted a release change. woot!